src/transaction/lock_manager.c

```diff
@@ -2921,7 +2941,7 @@ lock_grant_blocked_waiter_partial (THREAD_ENTRY * thread_p, LK_RES * res_ptr, LK
  *
  */
 static bool
-lock_check_escalate (THREAD_ENTRY * thread_p, LK_ENTRY * class_entry, LK_TRAN_LOCK * tran_lock)
+lock_check_escalate (THREAD_ENTRY * thread_p, const LK_ENTRY * class_entry, const LK_TRAN_LOCK * tran_lock)
```

Comment:
These `const` additions are also not needed for the proper functioning of the test and I am willing to remove them.
Investigating the CUBRID codebase is hard giving that const correctness is not made use of and the vast majority of the pointer arguments of the functions which could be `pointer to const` are not.